bindata/v4.1.0/kube-controller-manager/pod.yaml

```diff
@@ -87,6 +83,9 @@ spec:
     command: ["/bin/bash", "-ec"]
```

Comment:
Don't you want to change this one into `["/bin/bash", "-euxo", "pipefail", "-c"]` as well?

Reply:
done